terraform/alerting/alerts.tf

```diff
@@ -35,49 +35,53 @@ locals {
     ]
   ))
 
-  forward_progress_indicators = merge({
```

Comment:
Review this with https://github.com/google/exposure-notifications-verification-server/pull/2313/files?w=1